punchbowl/levelq/limits.py

```diff
@@ -0,0 +1,172 @@
+from collections.abc import Iterable
```

Comment:
@d-lamb you might find @svank 's implementation of limits helpful for outlier rejection at raw --> L0